### PR TITLE
Rename to ylem

### PR DESCRIPTION
**Pull request by imaustink**
Renames `react-view-model` and `RVM` to `ylem`

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-Thank you for contributing to React-View-Model!
+Thank you for contributing to ylem!
 
-If you need any help setting up a CanJS development environment and fixing React-View-Model bugs, please reach out to us on the [canjs/canjs Gitter channel](https://gitter.im/canjs/canjs) or email (contact@bitovi.com).  We will happily walk you through setting up a the environment, creating a test, and submitting a pull request.
+If you need any help setting up a CanJS development environment and fixing ylem bugs, please reach out to us on the [canjs/canjs Gitter channel](https://gitter.im/canjs/canjs) or email (contact@bitovi.com).  We will happily walk you through setting up a the environment, creating a test, and submitting a pull request.
 
 ## Reporting Bugs
 
-To report a bug, please visit [GitHub Issues](/canjs/react-view-model/issues).
+To report a bug, please visit [GitHub Issues](/canjs/ylem/issues).
```
Comment (christopherjbaker) [inline contributing.md]: should be `/bitovi/ylem`

File: contributing.md [hunk 2]
```diff
@@ -14,14 +14,14 @@ Search for previous tickets, if there is one add to that one rather than creatin
...
-1. Navigate to your clone of the React-View-Model repository - `cd /path/to/react-view-model`
+1. Navigate to your clone of the ylem repository - `cd /path/to/ylem`
 2. Create a new feature branch - `git checkout -b <branch name>`
 3. Make some changes
 4. Update tests to accomodate your changes
 5. Run tests and make sure they pass in all browsers
 6. Update documentation if necessary
 7. Push your changes to your remote branch - `git push origin <branch name>`
-8. Submit a pull request! Navigate to [Pull Requests](/canjs/react-view-model/pulls) and click the 'New Pull Request' button. Fill in some details about your potential patch including a meaningful title. When finished, press "Send pull request". The team will be notified about your submission and let you know of any problems or targeted release date.
+8. Submit a pull request! Navigate to [Pull Requests](/canjs/ylem/pulls) and click the 'New Pull Request' button. Fill in some details about your potential patch including a meaningful title. When finished, press "Send pull request". The team will be notified about your submission and let you know of any problems or targeted release date.
```
Comment (christopherjbaker) [inline contributing.md]: again

File: package.json [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-  "version": "2.0.0-pre.9",
+  "name": "ylem",
+  "version": "0.0.1",
   "description": "Add Observable View-Models to React Components",
   "homepage": "http://canjs.com",
   "repository": {
     "type": "git",
-    "url": "git://github.com/canjs/react-view-model.git"
+    "url": "git+https://github.com/bitovi/ylem.git",
+    "github": "https://github.com/canjs/canjs"
```
Comment (christopherjbaker) [inline package.json]: github should be same as url in this case

File: package.json [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
   },
   "author": {
     "name": "Bitovi",
     "email": "contact@bitovi.com",
-    "url": "http://bitovi.com"
+    "url": "http://bitovi.com",
+    "github": "https://github.com/canjs/canjs"
```
Comment (christopherjbaker) [inline package.json]: and again

File: package.json [hunk 5]
```diff
@@ -34,7 +36,12 @@
     "canjs-plugin",
     "donejs",
     "react",
-    "viewmodel"
+    "viewmodel",
+    "ylem",
+    "react-view-model",
```
Comment (christopherjbaker) [inline package.json]: no

File: package.json [hunk 6]
```diff
@@ -34,7 +36,12 @@
...
+    "rvm",
```
Comment (christopherjbaker) [inline package.json]: no

File: package.json (hunk 5 above)
Comment (imaustink) [reply package.json]: Yes

File: package.json (hunk 6 above)
Comment (imaustink) [reply package.json]: Also yes

**Comment by imaustink** [comment]
@christopherjbaker all good?

**Pull request merged**